portality/view/publisher.py

```diff
@@ -92,16 +93,26 @@ def upload_file():
     # file upload takes precedence over URL, in case the user has given us both
     if f.filename != "" and url is not None and url != "":
         flash("You provided a file and a URL - the URL has been ignored")
-    
+
+    try:
+        job = IngestArticlesBackgroundTask.prepare(current_user.id, upload_file=f, schema=schema, url=url, previous=previous)
+        IngestArticlesBackgroundTask.submit(job)
+    except BackgroundException as e:
+        flash(e.message, "error")
+        return render_template('publisher/uploadmetadata.html', previous=previous)
+
     if f.filename != "":
-        return _file_upload(f, schema, previous)
+        flash("File uploaded and waiting to be processed. Check back here for updates.", "success")
+        return render_template('publisher/uploadmetadata.html', previous=previous)
     
     if url is not None and url != "":
-        return _url_upload(url, schema, previous)
+        flash("File reference successfully received - it will be processed shortly", "success")
+        return render_template('publisher/uploadmetadata.html', previous=previous)
     
     flash("No file or URL provided", "error")
     return render_template('publisher/uploadmetadata.html', previous=previous)
 
+"""
```

Comment:
Is this code supposed to be commented out? It is actually modified below in a couple of places but the whole function is inactive.